src/swagger/schemas/common.yml

```diff
@@ -257,3 +257,23 @@ area-of-effect-model:
     type:
       type: string
       enum: [sphere, cone, cylinder, line, cube]
+damage-at-character-level-model:
+  description: |
+    'Spell Damage'
+  type: object
+  properties:
+    damage_at_character_level:
+      type: object
+      additionalProperties: true
+    damage_type:
+      $ref: './combined.yml#/APIReference'
+damage-at-slot-level-model:
+  description: |
+    'Spell Damage'
+  type: object
+  properties: 
+    damage_at_slot_level:
+      type: object
+      additionalProperties: true
+    damage_type:
+      $ref: './combined.yml#/APIReference'
```

Comment:
Can you put these in `spell.yml` for now? I don't think these shapes get used outside of Spells for right now.